config/Dockerfiles/tests.d/libvirt/02_libvirt-storage

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -xe
+
+# Verify basic provisioning for all supplied providers
+# distros.exclude: fedora26 centos7 fedora28
+# providers.include: libvirt
+# providers.exclude: none
+
+## NOTE: This is a temporary test to run libvirt tests until we fix
+## the networking race condition. Tests will only run on centos7.
+
+DISTRO=${1}
+PROVIDER=${2}
+
+TEST_NAME="${DISTRO}/${PROVIDER}/02_libvirt-storage"
+DESCRIPTION="Test ${PROVIDER} provider with storage provisioning"
+
+echo "Test Name: ${TEST_NAME}"
+echo "${DESCRIPTION}"
+
+TARGET="libvirt-storage"
+
+pushd docs/source/examples/workspace
+
+function clean_up {
+    set +e
+    #linchpin -w . -p PinFile.${PROVIDER}.yml -v destroy ${TARGET}
+    # remove output file used to gen uhash if it exists
+    rm storage.txt || true
+}
+trap clean_up EXIT SIGHUP SIGINT SIGTERM
+
+linchpin -w . -p PinFile.${PROVIDER}.yml -v up ${TARGET} 2> storage.txt
+
+cat storage.txt
+uhash=$(cat storage.txt | grep ${TARGET} | tail -n1 | cut -f3)
```

Comment:
`cat` is not needed here. 

`uhash=$(grep ${TARGET} storage.txt | tail -n1 | cut -f3 )` would give the same result.